dask/dataframe/core.py

```diff
@@ -1024,7 +1024,7 @@ def fillna(self, value=None, method=None, limit=None, axis=None):
             # Control whether or not dask's partition alignment happens.
             # We don't want for a pandas Series.
             # We do want it for a dask Series
-            if is_series_like(value):
+            if isinstance(value, pd.Series):
```

Comment:
```suggestion
            if is_series_like(value) and not is_dask_collection(value):
```

How about this then?

Reply:
Thanks, this works!